documentation/Functions.md

```diff
@@ -240,3 +240,12 @@ image: Path to an image to be displayed in the katacoda step.
 adaptTemplatesCobiGen()
 
 ***
+
+### executeFile
+#### parameter
+* directory to the script file, relative to workspace 
```

Comment:
For the other commands, we use numbers to specify the parameters.
And the file does not have to be a script file. It could also be an executable file (e.g. .exe, .msi)